contributing/ci-introduction.txt

```diff
@@ -62,21 +62,21 @@ Multiple labels are used in the PR reviewing process:
 Job names
 ^^^^^^^^^
 
-All core OME jobs are named after the following conventions
-``COMPONENT-VERSION-TYPE-DESCRIPTION`` where:
-
-- ``COMPONENT`` describes to core OME component, e.g. OMERO or BIOFORMATS.
-- ``VERSION`` describes to the MINOR.MINOR version, e.g. 5.0 or 5.1.
-- ``TYPE`` describes the source of the job and can be:
-
-  * the ``latest`` jobs build from the tip of the development branch, e.g.
-    `origin/dev_5_0`,
-  * the ``merge`` jobs build from the tip of the development branch with all
-    PRs merged using :ref:`scc merge` with the `org` default filter set,
-  * the ``breaking`` jobs build from the tip of the development branch with
-    all PRs labeled as ``breaking`` merged using :ref:`scc merge`,
-  * the ``release`` jobs build from and optionally create a tag at the tip of
+All core OME job names take the form
+``COMPONENT-VERSION-TYPE-DESCRIPTION``, where:
+
+- ``COMPONENT`` is the core OME component, e.g. OMERO or BIOFORMATS.
```

Comment:
BIOFORMATS -> Bio-Formats please. Otherwise this is good to merge, looks fine on staging thanks.